src/main/java/com/lgtm/easymoney/controllers/BalanceController.java

```diff
@@ -0,0 +1,56 @@
+package com.lgtm.easymoney.controllers;
+
+import com.lgtm.easymoney.payload.BalanceReq;
+import com.lgtm.easymoney.payload.BalanceRsp;
+import com.lgtm.easymoney.repositories.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping("/balance")
+public class BalanceController {
+    private final UserRepository userRepository;
+
+    @Autowired
+    public BalanceController(UserRepository userRepository) {
+        this.userRepository = userRepository;
+    }
+
+    @PutMapping("/update")
+    public ResponseEntity<?> update(@Valid @RequestBody BalanceReq balanceReq) {
+        var userWrapper = userRepository.findById(balanceReq.getUid());
```

Comment:
why do we need a wrapper? is it because it's `Repository` type and it's the expected usage?

Reply:
The return type of findById is Optional<User>